src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const RESET_DEFAULT_VALUE = 0;
+
+  const calculator = {
+    result: 0,
+
+    operate(actionCallback, value) {
+      actionCallback(value, this);
+
+      return this;
+    },
+
+    add(value, obj = this) {
```

Comment:
Do we need a default value here?

Reply:
if we're using operate method to call those functions - no, but if we want to use it without operate method it's needed